build-utils/css/utils/generateCSS.ts

```diff
@@ -58,6 +58,11 @@ export const generateNightThemeContent = async <
   const fullContent = `
   [data-theme='night'] {
     ${content}
+
+    /* Fix for dark mode surface hover color. Dark mode surfaces have a lighter color than day mode inverted to see shadows better. */
+    .elevated {
+      --amino-hover-color: var(--amino-gray-100);
+    }
```

Comment:
What do you think about using the theme variable here 🤔 
```suggestion
    /* Fix for dark mode surface hover color. Dark mode surfaces have a lighter color than day mode inverted to see shadows better. */
    .elevated {
      --amino-hover-color: ${theme.gray100};
    }
```